frontend/packages/console-app/src/components/quick-starts/QuickStartPanelContent.tsx

```diff
@@ -0,0 +1,72 @@
+import * as React from 'react';
+import * as classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import {
+  DrawerPanelContent,
+  DrawerPanelBody,
+  DrawerHead,
+  DrawerActions,
+  DrawerCloseButton,
+  Title,
+} from '@patternfly/react-core';
+import { QuickStart } from './utils/quick-start-types';
```

Comment:
nit: absolute import statements should be above others

Reply:
updated